xun/init/cookiecutter-workflow/cookiecutter.json

```diff
@@ -0,0 +1,4 @@
+{
+    "project_name": null,
+    "make_runnable": null
```

Comment:
This could have a default. But is it possible to make cookiecutter give you two choices?

Reply:
It is possible to give two choices, but it will then look like this:
```
module_name: m_1
Select make_runnable:
1 - yes
2 - no
Choose from 1, 2 [1]: 1
```